src/bin/miri.rs

```diff
@@ -362,6 +363,9 @@ fn main() {
                 "-Zmiri-tag-raw-pointers" => {
                     miri_config.tag_raw = true;
                 }
+                "-Zmiri-disable-weak-memory-emulation" => {
+                    miri_config.weak_memory_emulation = false;
```

Comment:
Please add the new flag to the README (and mention in the disable-data-race-detector docs that it also subsumes this flag).